code/game/machinery/_machinery.dm

```diff
@@ -107,9 +107,9 @@
 		//1 = use idle_power_usage
 		//2 = use active_power_usage
 	///the amount of static power load this machine adds to its area's power_usage list when use_power = IDLE_POWER_USE
-	var/idle_power_usage = 0
+	var/idle_power_usage = 100
 	///the amount of static power load this machine adds to its area's power_usage list when use_power = ACTIVE_POWER_USE
-	var/active_power_usage = 0
+	var/active_power_usage = 1000
```

Comment:
I'm not gonna go over the rest yet, cause I think this is something important to discuss

What is a power usage?
How much is "A lot"? How much is "A bit"?
How is a new "user" (Coder) supposed to figure these things out?

I would really recommend using the pattern arcane created for economy, where all values are based off a set of defines, and multipliers.

So active_power_usage `MID_DRAW * 4`, something like that.

Otherwise we will have drift, as the code changes and churns people will functionally undo your work, because the system they're working in doesn't guide them to the situation you want.
They will just do what they do currently, and pick random numbers that "look" right.

Reply:
Will do that